src/ConfigManager.h

```diff
@@ -116,6 +123,11 @@ class ConfigStringParameter : public BaseParameter {
         json->set(name, ptr);
     }
 
+    void clearData() {
+        memset(ptr, NULL, length);
+    }
```

Comment:
removing `strncpy(ptr, "", length - 1);` appears to work ... so outside of my comfort zone